cli.js

```diff
@@ -11,19 +11,54 @@ const cli = meow(`
 
 	Options
 	  --border-color  Color of the box border [black|red|green|yellow|blue|magenta|cyan|white|gray]
+	  --border-style  Style of the border box [single|double|round|single-double|double-single]
+	                  Can also be passed as a string specifying the characters to use. See example below.
 	  --padding       Space between the text and box border
 	  --margin        Space around the box
 
-	Example
+	Examples
 	  $ boxen I ❤ unicorns
 	  ┌────────────┐
 	  │I ❤ unicorns│
 	  └────────────┘
-`);
+
+	  $ boxen --border-style="double-single" …like everyone
+	  ╒══════════════╕
+	  │…like everyone│
+	  ╘══════════════╛
+
+	  $ boxen --border-style '1234-|' ASCII ftw!
+	  1----------2
+	  |ASCII ftw!|
+	  3----------4
+
+`, {
+	string: 'border-style'
+});
 
 const input = cli.input;
 
+function parseBorderStyle(cli) {
+	const borderStyle = cli && cli.flags && cli.flags.borderStyle;
+	const namedStyles = ['single', 'double', 'round', 'single-double', 'double-single'];
+	if (borderStyle && namedStyles.indexOf(borderStyle) === -1) {
+		if (borderStyle.length !== 6) {
+			console.error('Passed custom border-style is not valid');
+			process.exit(1);
+		}
+		cli.flags.borderStyle = {
+			topLeft: borderStyle[0],
+			topRight: borderStyle[1],
+			bottomLeft: borderStyle[2],
+			bottomRight: borderStyle[3],
+			horizontal: borderStyle[4],
+			vertical: borderStyle[5]
+		};
+	}
+}
+
 function init(data) {
+	parseBorderStyle(cli);
```

Comment:
Not a big fan of mutating functions. Instead accept the cli.borderStyle option and return a cleaned up one:

```js
cli.flags.borderStyle = cleanupBorderStyle(cli.flags.borderStyle);
```